CMakeLists.txt

```
@@ -825,6 +825,12 @@ if(CAPSTONE_BUILD_SHARED_LIBS)
         $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     )
     target_compile_definitions(capstone PUBLIC CAPSTONE_SHARED)
+    # Build pdb file for dll on Windows
+    if(WIN32)
```

```suggestion
    if(MSVC)
```

Almost forgot. This is the most reliable way to check for Windows, the `WIN32` had funny side effects IIRC.